Word-document/Merge-documents-in-same-page/.NET/Merge-documents-in-same-page/Program.cs

```diff
@@ -10,12 +10,12 @@ class Program
     {
         static void Main(string[] args)
         {
-            using (FileStream sourceStreamPath = new FileStream(Path.GetFullPath(@"../../../Data/SourceDocument.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+            using (FileStream sourceStreamPath = new FileStream(Path.GetFullPath(@"Data/SourceDocument.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
             {
                 //Opens an source document from file system through constructor of WordDocument class.
                 using (WordDocument sourceDocument = new WordDocument(sourceStreamPath, FormatType.Automatic))
                 {
-                    using (FileStream destinationStreamPath = new FileStream(Path.GetFullPath(@"../../../Data/DestinationDocument.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+                    using (FileStream destinationStreamPath = new FileStream(@"Data/DestinationDocument.docx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
```

Comment:
Why the Path.GetFullPath is removed here. Please do changes properly

Reply:
Addressed the concerns